Services/Store/StoreAdvancedConfig.php

```diff
@@ -70,6 +72,14 @@ public function getSyncDequeueSize()
         return $this->configManager->getStoreConfig(self::SYNC_DEQUEUE_SIZE, $this->storeId);
     }
 
+    public function hasToAddProductsInSyncOnAttributeSave()
+    {
+        return $this->configManager->getStoreConfig(
```

Comment:
return $this->configManager->getStoreConfig(
            self::HAS_TO_ADD_PRODUCTS_IN_SYNC_ON_ATTRIBUTE_SAVE,
            $this->storeId
        ) == 1;